spec/models/cavc_remand_spec.rb

```diff
@@ -0,0 +1,135 @@
+# frozen_string_literal: true
+
+describe CavcRemand do
+  describe ".create!" do
+    subject { CavcRemand.create!(params) }
+
+    let(:created_by) { create(:user) }
+    let(:updated_by) { create(:user) }
+    let(:appeal) { create(:appeal) }
+    let(:cavc_docket_number) { "123-1234567" }
+    let(:represented_by_attorney) { true }
+    let(:cavc_judge_full_name) { Constants::CAVC_JUDGE_FULL_NAMES.first }
+    let(:cavc_decision_type) { Constants::CAVC_DECISION_TYPES.keys.first }
+    let(:remand_subtype) { Constants::CAVC_REMAND_SUBTYPES.keys.first }
+    let(:decision_date) { 5.days.ago.to_date }
+    let(:judgement_date) { 4.days.ago.to_date }
+    let(:mandate_date) { 3.days.ago.to_date }
+    let(:decision_issues) do
+      create_list(
+        :decision_issue,
+        3,
+        :rating,
+        decision_review: appeal,
+        disposition: "denied",
+        description: "Decision issue description",
+        decision_text: "decision issue"
+      )
+    end
+    let(:decision_issue_ids) { decision_issues.map(&:id) }
+    let(:instructions) { "Intructions!" }
+
+    let(:params) do
+      {
+        created_by: created_by,
+        updated_by: updated_by,
+        appeal: appeal,
+        cavc_docket_number: cavc_docket_number,
+        represented_by_attorney: represented_by_attorney,
+        cavc_judge_full_name: cavc_judge_full_name,
+        cavc_decision_type: cavc_decision_type,
+        remand_subtype: remand_subtype,
+        decision_date: decision_date,
+        judgement_date: judgement_date,
+        mandate_date: mandate_date,
+        decision_issue_ids: decision_issue_ids,
+        instructions: instructions
+      }
+    end
+
+    it "creates the record" do
+      expect { subject }.not_to raise_error
+      params.each_key { |key| expect(subject.send(key)).to eq params[key] }
+    end
+
+    context "when missing required attributes" do
```

Comment:
Improving consistency (at least within this file)
```suggestion
    context "When missing required attributes" do
```